package.json

```diff
@@ -16,9 +16,11 @@
     "node": "6.x"
   },
   "dependencies": {
+    "bootstrap": "^4.0.0-alpha.5",
     "bootstrap-sass": "^3.3.7",
     "classnames": "^2.2.5",
     "dropzone": "^4.2.0",
+    "font-awesome": "^4.7.0",
```

Comment:
It seems we're adding yet another symbol font and we use it for a single icon. GitHub has recently moved from symbol fonts to inline SVG, do we really need this?